exporters/fluentd/src/log/recordable.cc

```diff
@@ -5,6 +5,10 @@
 #include "opentelemetry/exporters/fluentd/common/fluentd_common.h"
 #include "opentelemetry/exporters/fluentd/common/fluentd_logging.h"
 
+#include "opentelemetry/logs/severity.h"
```

Comment:
Should we document the minimum main repo version somewhere?

Reply:
Yes, I plan to add tag on top of commit after this PR, and document the mapping between main repo version and the github tag.